app/src/androidTest/java/org/mozilla/fenix/ui/robots/ThreeDotMenuMainRobot.kt

```diff
@@ -207,12 +207,13 @@ class ThreeDotMenuMainRobot {
                 waitingTime
             )
 
-            collectionNameTextField().check(matches(hasFocus()))
             collectionNameTextField().perform(
                 ViewActions.replaceText(name),
                 ViewActions.pressImeActionButton()
             )
 
+            mDevice.waitNotNull(Until.gone(By.res("org.mozilla.fenix.debug:id/createCollectionWrapper")))
```

Comment:
Let's add a comment above in case the design ever changes this would need to be changed.

Reply:
done